frontend/src/organisms/Navigation.js

```diff
@@ -1,50 +1,67 @@
 import React from 'react';
 
-import Navbar from "react-bootstrap/Navbar";
-import Nav from "react-bootstrap/Nav";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav';
+import Button from 'react-bootstrap/Button';
+import { SignInPage } from '../pages/SignInPage';
+import { SignUpPage } from '../pages/SignUpPage';
+import { ForgottenPasswordPage } from '../pages/ForgottenPasswordPage';
 
 import { Link } from 'react-router-dom';
 import { useAuth } from 'src/utils/auth';
 import { useHistory } from 'react-router-dom';
 import { route } from 'src/Routes';
 
 export function Navigation() {
-  const {user, signout } = useAuth();
+  const { user, signout } = useAuth();
   const history = useHistory();
 
   const homeLink = route.home();
   const profileLink = route.userProfile();
-  const signInLink = route.signIn();
-  const signUpLink = route.signUp();
+
+  const [showSignUp, setSignUpVisible] = useState(false);
+  const [showSignIn, setSignInVisible] = useState(false);
+  const [showForgotten, setForgottenVisible] = useState(false);
+
 
   return (
-    <Navbar collapseOnSelect expand="lg" bg="light" variant="light" className="navbar">
-      <Navbar.Brand as={Link} to={homeLink}>Fit.me</Navbar.Brand>
-      <Navbar.Toggle aria-controls="responsive-navbar-nav" />
-      <Navbar.Collapse id="responsive-navbar-nav">
-      <Nav className="mr-auto">
-        <Nav.Link as={Link} to="/about">O aplikaci</Nav.Link>
-     </Nav>
-     {user ? (
-       <Nav>
-        <Nav.Link as={Link} to={profileLink}>Profil</Nav.Link>
-        <Nav.Link onClick={() => {
-            signout();
-            history.push(route.home());
-            window.location.reload();
-          }}
-        >
-          Odhlásit se
-        </Nav.Link>
-       </Nav>
-      ) : (
-        <Nav>
-          <Nav.Link as={Link} to={signInLink}>Přihlášení</Nav.Link>
-          <Nav.Link as={Link} to={signUpLink}>Registrace</Nav.Link>
-        </Nav>
-      )
-     }
-      </Navbar.Collapse>
-    </Navbar>
+    <>
+      <Navbar collapseOnSelect expand="lg" bg="light" variant="light" className="navbar">
+        <Navbar.Brand as={Link} to={homeLink}>Fit.me</Navbar.Brand>
+        <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+        <Navbar.Collapse id="responsive-navbar-nav">
+          <Nav className="mr-auto">
+            <Nav.Link as={Link} to="/about">O aplikaci</Nav.Link>
+          </Nav>
+          {user ? (
+            <Nav>
+              <Nav.Link as={Link} to={profileLink}>Profil</Nav.Link>
+              <Nav.Link onClick={() => {
+                signout();
+                history.push(route.home());
```

Comment:
pokud tu nepoužíváš ty const vytvořené nahoře, tak je smázni

Reply:
to je smazaný